pkg/scheduler/queue/user_queues.go

```diff
@@ -25,15 +25,15 @@ type querier struct {
 	disconnectedAt time.Time
 }
 
-// This struct holds user queues for pending requests. It also keeps track of connected queriers,
-// and mapping between users and queriers.
-type queues struct {
-	userQueues map[string]*userQueue
+// This struct holds tenant queues for pending requests. It also keeps track of connected queriers,
+// and mapping between tenants and queriers.
+type tenantQueues struct {
```

Comment:
Not: should the file be renamed as well?

Reply:
Good point!